### DISCUSSION TITLE
点歌：强化学习相关（特别是AlphaGo/AlphaGo Zero/AlphaZero/MuZero系列）

### DISCUSSION THREAD

**Discussion by Jazihars** [mli/paper-reading#160] [General]
非常期待老师讲一下强化学习相关的知识，很想听AlphaGo/AlphaGo Zero/AlphaZero/MuZero系列的论文。

如果老师愿意专门开一门课讲强化学习的话最好了！

**Comment by BIGBALLON** [comment]
> 非常期待老师讲一下强化学习相关的知识，很想听AlphaGo/AlphaGo Zero/AlphaZero/MuZero系列的论文。
> 
> 如果老师愿意专门开一门课讲强化学习的话最好了！

提供一个很久以前的学习资料 （各位远程组会朋友们可以随意参考）

https://github.com/BIGBALLON/Toward-AGZ
